src/main/java/at/hannibal2/skyhanni/features/combat/BestiaryData.kt

```diff
@@ -448,26 +456,21 @@ object BestiaryData {
         return false
     }
 
-    enum class FormatType(val type: String) {
-        SHORT("Short"),
-        LONG("Long")
-    }
-
     enum class NumberType(val type: String) {
         INT("Normal (1, 2, 3)"),
         ROMAN("Roman (I, II, III)")
     }
 
-    enum class DisplayType(val type: String) {
-        GLOBAL_MAX("Global display (to max)"),
-        GLOBAL_TIER("Global display (to next tier)"),
-        LOWEST_TOTAL("Lowest total kills"),
-        HIGHEST_TOTAL("Highest total kills"),
-        LOWEST_NEEDED_MAX("Lowest kills needed to max"),
-        HIGHEST_NEEDED_MAX("Highest kills needed to max"),
-        LOWEST_NEEDED_TIER("Lowest kills needed to next tier"),
-        HIGHEST_NEEDED_TIER("Highest kills needed to next tier"),
-    }
+//     enum class DisplayType(val type: String) {
```

Comment:
Add a note why we're commenting it out, or remove?

Reply:
forgot to remove